doc/nrf/gs_programming.rst

```diff
@@ -11,11 +11,17 @@ the Nordic Edition of the SEGGER Embedded Studio (SES) IDE.
 	If you prefer to build your applications from the command line,
 	see :ref:`zephyr:getting_started_run_sample`.
 
+.. note::
+
+   See :ref:`programming pre-compiled firmware images<precompiled_fw>` and :ref:`building and programming sample application <building_pgming>` for information about programming a Thingy:91.
+
 .. _gs_programming_ses:
 
 Building with SES
 *****************
 
+.. build_SES_projimport_open_start
+
 Complete the following steps to build |NCS| projects with SES after :ref:`installing SEGGER Embedded Studio <installing_ses_win>` and :ref:`completing the first time setup <build_environment_win>`:
 
 1. Run the file :file:`bin/emStudio`.
```

Comment:
The sentence sounds like the user is supposed to run a script of some sort. Should it be "Start Segger Embedded Studio by running 'bin/emStudio' "?

Reply:
Modified as 
```suggestion
1. Start SEGGER Embedded Studio by running :file:`bin/emStudio`.
```